spec/index.bs

```diff
@@ -6567,7 +6567,9 @@ pipeline, and have the following members:
                 1. Let |limits| be |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.
                 1. Let |bindGroupLayouts| be a copy of |descriptor|.{{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
                 1. For each |i| in the [=list/get the indices|indices=] of |bindGroupLayouts|:
-                    1. If |bindGroupLayouts|[|i|] is `undefined` or [=list/empty=], set |bindGroupLayouts|[|i|] to `null`.
+                    1. If |bindGroupLayouts|[|i|] is `undefined` or
```

Comment:
I just realized this didn't quite make sense either:
- This would "crash" if `bindGroupLayouts[i]` is `null` (and not `undefined`).
- `bindGroupLayouts[i]` can't be `undefined`, only `null`. WebIDL converts any `undefined` values in JS into `null` values in step 3: https://webidl.spec.whatwg.org/#js-nullable-type

Does this seem right to you?

```suggestion
                    1. If |bindGroupLayouts|[|i|] is not `null` and
```

Reply:
LGTM. Thanks for checking!